runtime/shared/shrclssup.c

```diff
@@ -145,6 +145,13 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void* reserved)
 					runtimeFlags &= (~J9SHR_RUNTIMEFLAG_ENABLE_SHAREANONYMOUSCLASSES);
 				}
 
+				/* Check for -XX:+ShareUnsafeClasses and -XX:-ShareUnsafeClasses; whichever comes later wins. Enable is set by default so we just need to disable when that's the case. */
+				argIndex1 = FIND_ARG_IN_VMARGS(OPTIONAL_LIST_MATCH, VMOPT_XXENABLESHAREUNSAFECLASSES, NULL);
```

Comment:
These checks need to consume the args, see https://github.com/eclipse/openj9/issues/6870#issuecomment-529021381

Reply:
Thank you for the information! I will also change the error message later.

Reply:
Pls explain. What error message do you plan to change?

Reply:
According to Jack's comment, I will change the error message of `Command-line option unrecognised:` into some thing like `.... should only be used with -Xshareclasses option` if these options are used without `-Xshareclasses`

Reply:
I don't think we should do that. What we can do is quietly consume the options if -Xshareclasses doesn't do it. See https://github.com/eclipse/openj9/blob/dde4b86e24f7e01f8f70c8d16bde45b0cb4efdd5/runtime/vm/jvminit.c#L2541